mail_activity/views/mail_activity.xml

```diff
@@ -0,0 +1,181 @@
+            <form string="Activities">
+                <sheet>
+                    <div class="oe_title">
+                        <label for="name" class="oe_edit_only"/>
+                        <h1><field name="name"/></h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="category" invisible="1"/>
+                            <field name="res_model_id" groups="base.group_no_one"/>
+                            <field name="summary"/>
+                            <field name="icon" groups="base.group_no_one"/>
+                            <field name="days"/>
+                        </group>
+                        <group>
+                            <field name="next_type_ids" widget="many2many_tags"/>
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="mail_activity_type_view_tree" model="ir.ui.view">
+        <field name="name">mail.activity.type.view.tree</field>
+        <field name="model">mail.activity.type</field>
+        <field name="arch" type="xml">
+            <tree string="Activities">
+                <field name="sequence" widget="handle"/>
+                <field name="name"/>
+                <field name="summary"/>
+                <field name="days"/>
+                <field name="res_model_id" groups="base.group_no_one"/>
+                <field name="icon" groups="base.group_no_one"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="mail_activity_type_action" model="ir.actions.act_window">
+        <field name="name">Activity Types</field>
+        <field name="res_model">mail.activity.type</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <menuitem
+      id="menu_mail_activity_type"
+      action="mail_activity_type_action"
+      parent="base.menu_email"
+    />
+
+    <record id="mail_activity_view_form_popup" model="ir.ui.view">
+        <field name="name">mail.activity.view.form.popup</field>
+        <field name="model">mail.activity</field>
+        <field name="priority">20</field>
+        <field name="arch" type="xml">
+            <form string="Log an Activity" create="false">
+                <sheet string="Activity">
+                    <group invisible="1">
+                        <field name="activity_category" invisible="1" />
+                        <field name="res_model" invisible="1"/>
+                        <field name="res_model_id" invisible="1"/>
+                        <field name="res_id" invisible="1"/>
+                        <field name="previous_activity_type_id"/>
+                        <field name="has_recommended_activities"/>
+                    </group>
+                    <group attrs="{'invisible': [('has_recommended_activities','=',False)]}">
```

Comment:
I've set 1 recommended next activity on a type:

![image](https://user-images.githubusercontent.com/10028499/41642279-27d8f692-7468-11e8-8dce-3ddf22b6df93.png)

When I select the activity type:

![image](https://user-images.githubusercontent.com/10028499/41642348-4e53437c-7468-11e8-8ed2-ecb07b2d28b3.png)

The radio button doesn't seem to appear.

Reply:
I have not fixed. But updated the README stating as known issue.